css/src/icons.css

```diff
@@ -45,15 +45,16 @@
 .icon-taboola:before,
 .icon-tiktok:before,
 .icon-twitter:before,
-.icon-youtube:before {
+.icon-youtube:before,
+.icon-demographics:before {
     font-family: "ocelot-social-icon" !important;
```

Comment:
I guess this name isn't really accurate anymore. It's more `ocelot-icon-v2` than `ocelot-social-icon`. Not really asking for a change, more a consideration for the next time we're working in here.

Reply:
Yeah that's what I was thinking too. Let me see if I can change this without breaking anything.

Reply:
Updated.